schemas/v1.0/schema.yaml

```diff
@@ -1,5 +1,101 @@
-$id: https://spec.openapis.org/overlay/1.0/schema/WORK-IN-PROGRESS
+$id: https://spec.openapis.org/overlay/1.0/schema/2024-10-17
 $schema: https://json-schema.org/draft/2020-12/schema
-description: The description of Overlay v1.0.x documents
-
+description: The description of OpenAPI Initiative Overlay v1.0.0 documents
+  without schema validation, as defined by
+  https://spec.openapis.org/overlay/v1.0.0
 type: object
+$comment: https://spec.openapis.org/overlay/v1.0.0#overlay-object
+properties:
+  overlay:
+    description: The version number of the Overlay Specification
+    type: string
+    pattern: ^1\.0\.\d+(-.+)?$
```

Comment:
```suggestion
    pattern: ^1\.0\.\d+$
```

Reply:
why diverge from OAS consistency?  This pattern is used in OAS and Arazzo.

Reply:
OAS needs this pattern because they published `3.0.0-rc0` etc. and `3.1.0-rc0` etc.

Arazzo doesn't need it because they only published `1.0.0` so far.

Consistently and blindly copying a too generous pattern is not a practice I embrace.